cmd/evmosd/testnet.go

```diff
@@ -148,6 +160,18 @@ Example:
 			args.startingIPAddress, _ = cmd.Flags().GetString(flagStartingIPAddress)
 			args.numValidators, _ = cmd.Flags().GetInt(flagNumValidators)
 			args.algo, _ = cmd.Flags().GetString(flags.FlagKeyType)
+			baseFee, _ := cmd.Flags().GetString(flagBaseFee)
+			minGasPrice, _ := cmd.Flags().GetString(flagMinGasPrice)
+
+			ok := true
+			args.baseFee, ok = sdk.NewIntFromString(baseFee)
+			if !ok || !args.baseFee.IsPositive() {
+				return fmt.Errorf("invalid value for --base-fee. expected an uint but got %s", baseFee)
+			}
+			args.minGasPrice, err = sdk.NewDecFromStr(minGasPrice)
+			if err != nil || !args.minGasPrice.IsPositive() {
+				return fmt.Errorf("invalid value for --min-gas-price. expected an uint or positive decimal but got %s", minGasPrice)
+			}
```

Comment:
Nit: here I would separate the error messages to be more clear. I mean, if err != nil, return the error. And then, if minGasPrice < 0, return the corresponding error message

Reply:
In fact, I don’t think it’s necessary to write it so complicatedly. I believe that as a programmer, him should have a certain understanding that negative numbers cannot be used for these two parameters. Moreover, even if the genesis.json file is made with negative numbers, the chain cannot be started.
Of course, I've explained both situations separately.